### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by juler69** [ibmdb/node-ibm_db#731]
@juler69 To install ibm_db on Z/OS system, you need to first configure the ODBC driver on z/OS by following the steps documented here: https://github.com/ibmdb/node-ibm_db#configure-odbc-driver-on-zos
Then set the IBM_DB_HOME environment variable which should point to the HLQ of the Db2 datasets. Then install ibm_db. Setting IBM_DB_INSTALLER_URL environment variable requires a downloaded clidriver .zip or .tar.gz file. IBM_DB_INSTALLER_URL is not for installation on Z/OS system. Hope it'll help you. Thanks.

_Originally posted by @bimalkjha in https://github.com/ibmdb/node-ibm_db/issues/727#issuecomment-747576060_

**Comment by juler69** [comment]
The configuration steps for the driver are already in place.  I cannot install the package though so I can't use it.  The link you gave refers to the documentation I already followed to the point of my error.  The environment variable you say is not used on z/OS - why does the documentation not indicate this?  How then do I point to a downloaded driver on my z/OS filesystem and avoid the error where it is trying to download it?

Tried again using the git clone option and npm install - same issue and still seems to be trying to connect externally for something and is timing out.

> ibm_db@2.7.1 install /u/A123456/projects/mis-service-monitoring/node-ibm_db
> node installer/driverInstall.js

platform =  os390 , arch =  s390x , node.js version =  v12.18.0
make: Error -- FSUM8705 No target
Unable to find 'make' in PATH. Installation may fail!
oput 'SYS1.DB2T.SDSNMACS(DSNAO64C)' '/u/A123456/projects/mis-service-monitoring/node-ibm_db/build/dsnao64c.x'

Error: Command failed: node-gyp configure build  --IS_DOWNLOADED=false --IBM_DB_HOME="$IBM_DB_HOME"
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: connect ETIMEDOUT 104.20.22.46:443
gyp ERR! stack     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1147:16)
gyp ERR! System OS/390 26.00
gyp ERR! command "/usr/lpp/node-v12-os390-s390x/IBM/node-v12.18.0-os390-s390x/bin/node" "/usr/lpp/node-v12-os390-s390x/IBM/node-v12.18.0-os390-s390x/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build" "--IS_DOWNLOADED=false" "--IBM_DB_HOME=SYS1.DB2T"
gyp ERR! cwd /u/P719952/projects/mis-service-monitoring/node-ibm_db
gyp ERR! node -v v12.18.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1042:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node-gyp configure build  --IS_DOWNLOADED=false --IBM_DB_HOME="$IBM_DB_HOME"'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ibm_db@2.7.1 install: `node installer/driverInstall.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ibm_db@2.7.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /u/A123456/.npm/_logs/2021-01-10T23_35_01_743Z-debug.log
